internal/logger/logger.go

```diff
@@ -133,6 +133,10 @@ func Info(v ...interface{}) {
 	defaultInfoLogger.Println(v...)
 }
 
+func Fatal(v ...interface{}) {
```

Comment:
Can you please write a one-liner as like in the other exported method?

Also, can you please update the description how fatal logs look like?